### ISSUE TITLE
Reply to SMS, like in Plasma

### ISSUE THREAD

**Issue by souperdoupe** [Bajoja/indicator-kdeconnect#78]
Hey, I love this project because I'm not the biggest fan of KDE.  However, I noticed that I can't reply to SMS messages, like in this example: https://forum.manjaro.org/uploads/default/original/2X/e/ebe3d2b5478525f2b9fa6d7d5cbcde58a7febf81.png

I understand that the image above is default behavior in the plasma desktop environment, not with your app (https://forum.manjaro.org/t/kde-connect-v-1-where-are-sms-replay-abilities/10372).  But, I'm not sure if your app offers this feature, and if not, if you have any intent to update it for SMS reply features.  Consequently, I'm unsure whether to mark this as a bug or a suggestion.

This is a different issue from this one: https://github.com/Bajoja/indicator-kdeconnect/issues/76 <- I solved that issue by adding python3-gi, but this behavior was occurring before and after installing gi.

I'm running Debian stretch (testing).

**Comment by Bajoja** [comment]
Hi @souperdoupe, the question here is the fact that response SMS by that way already exists on the notification buble. There's a button to response, and if you click it it open a similiar window.
This only don't work on Ubuntu.

**Comment by souperdoupe** [comment]
Hey, I did some digging and apprently the issue I'm describing lies with kdeconnect itself, a bug that existed since October 2016.  I thought it was with your project, but I went ahead and tested it by installing plasma, uninstalling your app, and testing again -- which yielded the same results.  Not sure what the issue is, but it definitely has to do with the kdeconnect app itself.  Sorry to bring this up!
